pkg/controller/daemon/config/v1alpha1/register.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+var (
```

Comment:
I'm wondering if it would make sense to register the `addKnownTypes` and `addDefaultingFuncs` funcs to the schemebuilder by default so that every scheme & package becomes "independent" and can be added to an arbitrary scheme without any new things. The recommendeddefaults pattern would still apply but it'd also be called by a SetDefaults func...

Reply:
If need, will add it later
The key point is that the `--config` not implement in KCM. we can add at same time